circle.yml

```diff
@@ -252,8 +252,8 @@ jobs:
       - benchmark:
           min_time: "0.01"
       - spectest:
-          expected_passed: 4903
-          expected_failed: 529
+          expected_passed: 4997
```

Comment:
wow nice increase

My PRs are getting like 2 new tests passed :confounded:

Reply:
This is part of type validation, so I expect it has a lot of test cases.